.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+node_modules
```

Comment:
Why is there node_modules in the gitignore twice?

Reply:
That was an auto add by my Visual Studio Code. I will remove in next PR.